### PR TITLE
Minimum go version is 1.16

### PR DESCRIPTION
**Pull request by fgsch**
Spotted by @Integralist.

### CODE REVIEW

File: content/getting-started/install/macos.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ description: >
 
 This page has information on how to install and use TinyGo on macOS. If you wish to build TinyGo from source, for example if you intend to contribute to the project, please take a look [here](../../../docs/guides/build).
 
-You must have Go v1.15+ already installed on your machine in order to install TinyGo. We recommend Go v1.17+.
+You must have Go v1.16+ already installed on your machine in order to install TinyGo. We recommend Go v1.17+.
```
Comment (deadprogram) [inline content/getting-started/install/macos.md:12]: We probably should also "recommend" `v1.18+`

Comment (fgsch) [reply content/getting-started/install/macos.md:12]: I can change it but the other versions are recommending 1.17+.

Comment (deadprogram) [reply content/getting-started/install/macos.md:12]: Maybe we should change them all? Probably yes.

Comment (fgsch) [reply content/getting-started/install/macos.md:12]: Sure. I will do that in a separate PR though.

**Comment by deadprogram** [comment]
Thanks @fgsch

**Pull request merged**